fragments/accordion/hatch.js

```diff
@@ -6,9 +6,12 @@ module.exports = {
         //Header
     	f.addIf($.find('h2').first(), 'model.showtitle == \'true\'')
         f.mapField($.find('h2').first(), "model.title")
+        f.bindAttribute($.find('h2').first(), "data-per-inline", "`model.title`")
```

Comment:
this should not be necessary, please upgrade to the latest percli version - mapField and mapRichField are adding data-per-inline automatically in the latest version